### ISSUE TITLE
Voiceover unable to select stop from Map

### ISSUE THREAD

**Issue by ualch9** [OneBusAway/onebusaway-ios#526]
See attached.

![image](https://user-images.githubusercontent.com/22162410/133547116-3ec4234a-0cb1-4d57-9ae6-d8136073ac45.png)

-- image from direct user feedback
